velox/functions/prestosql/tests/ProbabilityTest.cpp

```diff
@@ -155,5 +155,63 @@ TEST_F(ProbabilityTest, binomialCDF) {
       binomialCDF(-2, 0.5, -1), "numberOfTrials must be greater than 0");
 }
 
+TEST_F(ProbabilityTest, invBinomialCDF) {
```

Comment:
Nit : Rename the test inverseBinomialCDF to make the function name explicit.

Reply:
Fixed in most recent commit `f2a1493`